pyproject.toml

```diff
@@ -20,57 +20,6 @@ include = [
     '/requirements',
 ]
 
-[project]
```

Comment:
Does this whole section need to be moved? It makes it harder to confirm the small diff, and will affect the git blame for this file for future reference.

Reply:
No, PDM did this automatically and I didn't want to interfere because until now I thought this change was in `pdm.lock` :laughing:

Reply:
let's revert all these changes.